cmd/limactl/list.go

```diff
@@ -73,8 +93,10 @@ func listAction(cmd *cobra.Command, args []string) error {
 	w := tabwriter.NewWriter(cmd.OutOrStdout(), 4, 8, 4, ' ', 0)
 	fmt.Fprintln(w, "NAME\tSTATUS\tSSH\tARCH\tCPUS\tMEMORY\tDISK\tDIR")
 
-	if len(instances) == 0 {
+	if len(allinstances) == 0 {
 		logrus.Warn("No instance found. Run `limactl start` to create an instance.")
+	} else if len(instances) == 0 {
```

Comment:
This will not show an error when some instances exist, but others don't:

```console
$ limactl ls foo bar
WARN[0000] No instance matching [foo bar] found.
NAME    STATUS    SSH    ARCH    CPUS    MEMORY    DISK    DIR
$ limactl ls foo bar alpine
NAME      STATUS     SSH                ARCH      CPUS    MEMORY    DISK      DIR
alpine    Running    127.0.0.1:51115    x86_64    4       4GiB      100GiB    /Users/jan/.lima/alpine
```

Reply:
```console
$ _output/bin/limactl list foo bar alpine
WARN[0000] No instance matching foo found.              
WARN[0000] No instance matching bar found.              
NAME      STATUS     SSH            ARCH      CPUS    MEMORY    DISK      DIR
alpine    Stopped    127.0.0.1:0    x86_64    4       4GiB      100GiB    /home/anders/.lima/alpine

```